src/main/resources/liquibase.xml

```diff
@@ -0,0 +1,61 @@
+<databaseChangeLog logicalFilePath="liquibase.xml"
```

Comment:
@riteshghiya90 we might also need `failed_event_retry_log` table. can you please refer https://github.com/Bahmni/crater-atomfeed repo

Reply:
We don't need this table as of now, since we are not retrying the failed events. We are just storing these failed events for reference. Actually we don't update the markers unless we successfully publish the event.